bitnami/kafka/templates/statefulset.yaml

```diff
@@ -188,6 +194,12 @@ spec:
             - name: KAFKA_OPTS
               value: "-Djava.security.auth.login.config=/opt/bitnami/kafka/conf/kafka_jaas.conf"
             {{- if (include "kafka.client.saslAuthentication" .) }}
+            {{- if and .Values.auth.jaas.clientUsers .Values.auth.jaas.clientPasswords }}
```

Comment:
Honestly, I would deprecate `.Values.auth.jaas.clientUser` and `.Values.auth.jaas.clientPassword`, we're overcomplicating the logic trying to maintaining both formats.